<?xml version="1.0" encoding="UTF-8"?>
<bajadoc version="2.0" createdBy="niagara-baja-doclet-1.0.9" createdAt="03-May-2024" createdOn="883e7f7a9875">
<class module="baja" runtimeProfile="rt" qualifiedName="javax.baja.sys.ActionInvokeException" name="ActionInvokeException" packageName="javax.baja.sys" public="true" category="exception">
<description>
The ActionInvokeException is an unchecked exception that&#xa; wraps an exception raised during an action invocation.
</description>
<tag name="@author">Brian Frank</tag>
<tag name="@creation">26 Jan 01</tag>
<tag name="@version">$Revision: 3$ $Date: 2/28/01 9:44:52 AM EST$</tag>
<tag name="@since">Baja 1.0</tag>
<extends>
<type class="javax.baja.sys.BajaRuntimeException"/>
</extends>
<!-- javax.baja.sys.ActionInvokeException(java.lang.String, java.lang.Throwable) -->
<constructor name="ActionInvokeException" public="true">
<parameter name="gripe">
<type class="java.lang.String"/>
</parameter>
<parameter name="ex">
<type class="java.lang.Throwable"/>
</parameter>
<description>
Constructor with specified message and target exception.
</description>
</constructor>

<!-- javax.baja.sys.ActionInvokeException(java.lang.Throwable) -->
<constructor name="ActionInvokeException" public="true">
<parameter name="ex">
<type class="java.lang.Throwable"/>
</parameter>
<description>
Constructor with specified message.
</description>
</constructor>

</class>
</bajadoc>
